backend/substrapp/views/utils.py

```diff
@@ -54,12 +52,7 @@ def node_has_process_permission(asset):
 
 
 class PermissionMixin(object):
-    authentication_classes = [
-        BasicAuthentication,  # for node to node
-        SecureJWTAuthentication,  # for user from front/sdk/cli
-        TokenAuthentication,  # for user from front/sdk/cli
-        CustomSessionAuthentication,  # for user on drf web browsable api
-    ]
+    authentication_classes = api_settings.DEFAULT_AUTHENTICATION_CLASSES + [BasicAuthentication]
```

Comment:
Could you explain the purpose of this change? `api_settings.DEFAULT_AUTHENTICATION_CLASSES` seems not equivalent to `(BasicAuthentication, TokenAuthentication)`.

Reply:
`api_settings.DEFAULT_AUTHENTICATION_CLASSES` will contain `SecureJWTAuthentication` and `ExpiryTokenAuthentication` (and `CustomSessionAuthentication` in dev mode). By adding BasicAuth to it, we're equivalent to the existing list when in dev mode and just miss the `CustomSessionAuthentication ` in prod mode.

There is one change though: `ExpiryTokenAuthentication` instead of `TokenAuthentication`. Not an issue since they are fully compatible, and we get the added benefit of checking the expiry date of the token (which we didn't, and I think this was a bug).

Reply:
Thanks for the explanation, yes my main concern was regarding the difference between `ExpiryTokenAuthentication` vs `TokenAuthentication`. As you said, it looks like a previous bug. LGTM.